frontend/common/src/components/Pending/Loading.tsx

```diff
@@ -1,17 +1,39 @@
 import React from "react";
 import { useIntl } from "react-intl";
 
-const Loading: React.FC = () => {
+export interface LoadingProps {
+  inline?: boolean;
+  /**
+   * Determine if the loading state should
+   * be announced to users
+   *
+   * This should be used sparingly and usually
+   * only with the inline={true}
+   *
+   * polite: wait for idle state (recommended)
+   * assertive: interrupt the current user workflow
+   *
+   * REF: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
+   */
+  live?: "polite" | "assertive";
+}
+
+const Loading: React.FC<LoadingProps> = ({ inline, live }) => {
```

Comment:
Is this the way we wanted to start doing this? (Happens to be no need to children here as well :) )

```suggestion
const Loading = ({ inline, live }: LoadingProps): JSX.Element => {
```

Reply:
Did we agree on adopting this pattern? I know we ran a poll but I wasn't aware of the outcome so I'm just following the established pattern. 

If we did, I can start updating all the components I touch going forward.